### PR TITLE
Added the find users belonging to a group query

### LINKED ISSUE #243: Get users who belong to the group query
## Feature Title

Get users who belong to the group.

## Acceptance Criteria

- [x] Added the query

## Technical Details

- API Changes:
- Database Changes:
- Dependencies:

## Additional Context

None

### PR DESCRIPTION
**Pull request by hitohata**
## Overviews of implementation

Added the find users belonging to a group query

## Review points

See the test

### CODE REVIEW

File: backend/main/src/Group/Query/FindUsersBelongingToAGroupQuery/FindUsersBelongingToAGroupQuery.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		for (const userData of usersData) {
+			if (userData) {
+				users.push({
+					id: userData.userId,
+					name: userData.username,
+				});
+			}
+		}
+
+		return Ok(users);
```
Comment (taniguchiiqqq) [inline backend/main/src/Group/Query/FindUsersBelongingToAGroupQuery/FindUsersBelongingToAGroupQuery.ts]: This comment is for confirmation.
Based on the API doc, the final data format should be `return Ok({users: users});`. But If my understanding is correct, we will format data in the router. Is that right?

**Review by taniguchiiqqq — APPROVED** [review]
Thank you! LGTM. Please answer to my comment, I wanna make sure my understanding.

File: backend/main/src/Group/Query/FindUsersBelongingToAGroupQuery/FindUsersBelongingToAGroupQuery.ts (hunk 1 above)
Comment (hitohata) [reply backend/main/src/Group/Query/FindUsersBelongingToAGroupQuery/FindUsersBelongingToAGroupQuery.ts]: No. Your understanding is incorrect.
This one comes from my overlooking.

I've fixed it. Thank you for pointing it out.

Comment (taniguchiiqqq) [reply backend/main/src/Group/Query/FindUsersBelongingToAGroupQuery/FindUsersBelongingToAGroupQuery.ts]: Thank you. I misunderstood it. Now I got it.

**Pull request merged**